src/Components/Notifications/NotificationsList.tsx

```diff
@@ -77,19 +105,21 @@ const NotificationsList: React.FC<NotificationsListProps> = ({
   }, [isMobile])
 
   const handleLoadNext = () => {
-    if (!relay.hasMore() || relay.isLoading()) {
+    if (!hasNext || isLoadingNext) {
       return
     }
 
-    setLoading(true)
-
-    relay.loadMore(10, err => {
-      if (err) console.error(err)
-
-      setLoading(false)
+    loadNext(10, {
+      onComplete: err => {
+        if (err) console.error(err)
+      },
     })
   }
 
+  // This is needed because `totalCount` and therefor `relay.hasMore()` doesn't work reliably
+  // TODO: Remove this once we have a reliable `totalCount`
+  const isLoading = isLoadingNext && nodes.length >= INITIAL_LOADING_SIZE
```

Comment:
**suggestion:** do we know how we can solve this? if so, it's worth it to ticket it in Jira and link it in code so we don't miss it.

Reply:
Let's create a ticket anyway so we can go back to this and find a solution.